index.html

```diff
@@ -1,657 +1,1295 @@
-                        </div>
-                        <div class="box-title-wrap" data-cmp-data-layer="" id="">
-                            <h4>Articles</h4>
-                            <p>Lorem ipsum dolor sit amet, consetetur sadip.</p>
-                        </div>
-                    </div>
-                    <div class="icon-box-item">
-                        <div data-asset-id="" id="" data-cmp-data-layer="" class="cmp-image" itemscope="" itemtype="">
-                            <img srcset="./assets/video.svg" src="./assets/video.svg" loading="lazy" class="logo-img" itemprop="contentUrl"  alt="logo" title="logo">
-                        </div>
-                        <div class="box-title-wrap" data-cmp-data-layer="" id="">
-                            <h4>Video Series</h4>
-                            <p>Lorem ipsum dolor sit amet, consetetur sadip.</p>
-                        </div>
+              </div>
+
+              <div class="container responsivegrid alert-btn-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                <div id="container-809c4a1b29" class="cmp-container">
+                  <div class="image">
+                    <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_465660961/container/image.coreimg{.width}.png/1669064796330/alert-img.png" data-asset="/content/dam/LeaderImpact/site-./assets/alert-img.png" data-title="logo" id="image-d2eb04f827" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                      <img src="./assets/alert-img.png" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="logo">
                     </div>
-                    <div class="icon-box-item">
-                        <div data-asset-id="" id="" data-cmp-data-layer="" class="cmp-image" itemscope="" itemtype="">
-                            <img srcset="./assets/book.svg" src="./assets/book.svg" loading="lazy" class="logo-img" itemprop="contentUrl"  alt="logo" title="logo">
-                        </div>
-                        <div class="box-title-wrap" data-cmp-data-layer="" id="">
-                            <h4>Books</h4>
-                            <p>Lorem ipsum dolor sit amet, consetetur sadip.</p>
+                  </div>
+                  <div class="button">
+                    <a id="button-571dd2b4d0" class="cmp-button" href="/content/leaderimpact/lang/en2/resources/impact-blog/what-s-on-the-mind-of-a-top-level-leader-.html">
+                      <span class="cmp-button__text">Learn More</span>
+                    </a>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid welcome aem-GridColumn aem-GridColumn--default--12 layout-grid" style="background-image:url(./assets/Welcome-bg.png);background-size:cover;background-repeat:no-repeat;">

```

Comment:
The background image does not go here, it goes on the line below with `cmp-container`.